examples/12. Advanced Action.md

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+
+# Advanced Action API
+An action in a resource in the API Blueprint can be considered as a state transition. This API example demonstrates that a resource can contain a state transition to another resource. This allows us to show that the resource `Task` can afford an action that is not limited to the resource `Task`.
+
+## API Blueprint
++ [Previous: Resource Model](11.%20Resource%20Model.md)
++ [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/11.%20Advanced%20Action.md)
+
+# Task [/tasks/{id}]
+
++ Parameters
+    + id (string)
+
+## Retrieve Task [GET]
+
++ Response 200 (application/json)
+
+        {
+            "id": 123,
+            "name": "Go to gym",
+            "done": false,
+            "type": "task"
+        }
+
+## Update Task [PATCH]
+This is a state transition within the same resource
```

Comment:
IMO, changing data does change state. There are infinite resource states that you can access using the `edit` transition. Those states all have the same affordances, but because of the resource has changed, it has a different state. From the state-machine standpoint, there are only so many combinations of a affordances, but depending what those affordances are, there are potentially infinite 'states' in any one of them (e.g. they allow you to edit). Make sense?